.travis.yml

```diff
@@ -37,10 +37,7 @@ env:
   - INCLUDE="test_module,second_module" UNIT_TEST="1" PG_LOGS_ENABLE="1" PG_NON_DURABILITY="1" 
   - VERSION="7.0" INCLUDE="test_module,second_module" ODOO_REPO="OCA/OCB" PG_LOGS_ENABLE="1" PG_NON_DURABILITY="1"  # ODOO_REPO usage example
   # - VERSION="6.1" INCLUDE="test_module,second_module"
-  - LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="27" TRAVIS_PULL_REQUEST="false" # Use main pylint config file
-  - VERSION="master" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="89" TRAVIS_PULL_REQUEST="true"  # Use PR pylint config file
-  - VERSION="7.0" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="82" TRAVIS_PULL_REQUEST="true"  # To check pylint_conf of PR's with old api
-  - VERSION="" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="27" TRAVIS_PULL_REQUEST="false"  # To check VERSION empty or unset case.
+  - LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="91"
```

Comment:
I don't get it, wich `def test_` are mean? in `self_test` I set the proper expected dict of errors to compare it, without getting the value `os.environ['PYLINT_EXPECTED_ERRORS']`.

`os.environ['PYLINT_EXPECTED_ERRORS']` is used only in test_pylint, because test_pylint is called from /travis/travis_run_tests